1-js/06-advanced-functions/06-function-object/article.md

```diff
@@ -241,34 +241,34 @@ let sayHi = function *!*func*/!*(who) {
 sayHi("John"); // Hello, John
 ```
 
-There are two special things about the name `func`, that are the reasons for it:
+Hay dos cosas especiales sobre el nombre `func`, que le hacen útil:
 
-1. It allows the function to reference itself internally.
-2. It is not visible outside of the function.
+1. Permite que la función se haga referencia internamente.
+2. No es visible fuera de la función..
 
-For instance, the function `sayHi` below calls itself again with `"Guest"` if no `who` is provided:
+Por ejemplo, la función `sayHi` a continuación se vuelve a llamar a sí misma con ` "Guest"  `si no se proporciona ` who`:
 
 ```js run
 let sayHi = function *!*func*/!*(who) {
   if (who) {
     alert(`Hello, ${who}`);
   } else {
 *!*
-    func("Guest"); // use func to re-call itself
+    func("Guest"); //usa func para volver a llamarse
```

Comment:
```suggestion
    func("Guest"); // usa func para volver a llamarse a sí misma
```
-creo que sin la aclaración parece que la llama uno
-está mal en inglés, 
it uses func to re-call itself